integrations/git_test.go

```diff
@@ -25,12 +25,17 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const (
-	littleSize = 1024              //1ko
-	bigSize    = 128 * 1024 * 1024 //128Mo
+var (
```

Comment:
If git files will not be affected, I don't think this is necessary?

Reply:
@lunny we use this for testing LFS.